app/src/createExcelFiles/fetchSearches.js

```diff
@@ -0,0 +1,25 @@
+import { SearchItem } from "./modal";
+import { mapToExcelArray } from "./ExcelUtils";
+
```

Comment:
your file name is inconsistent (camelCase here, while other places its not), so might wanna rename

Reply:
You have not done this yet, rename the file to be consistent elsewhere. @HassanChowdhry

Reply:
i did... for some reason it didnt update on github